tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.ts

```diff
@@ -88,62 +88,58 @@ class TfGraphDashboard extends LegacyElementMixin(PolymerElement) {
         trace-inputs="{{_traceInputs}}"
       ></tf-graph-controls>
       <div class="center" slot="center">
-        <template
-          is="dom-if"
-          if="[[_datasetsState(_datasetsFetched, _datasets, 'EMPTY')]]"
+        <tf-graph-dashboard-loader
+          id="loader"
+          datasets="[[_datasets]]"
+          selection="[[_selection]]"
+          selected-file="[[_selectedFile]]"
+          out-graph-hierarchy="{{_graphHierarchy}}"
+          out-graph="{{_graph}}"
+          out-stats="{{_stats}}"
+          progress="{{_progress}}"
+          hierarchy-params="[[_hierarchyParams]]"
+          compatibility-provider="[[_compatibilityProvider]]"
+        ></tf-graph-dashboard-loader>
+        <div
+          class$="no-data-message [[_showComponent(_selectedFile, _datasets, 'false')]]"
```

Comment:
Two related things.

1. Since we render `.no-data-message` and `.graphboard` all the time anyways, can we just set a class name on the outer DIV or whatever and use CSS to control whether to show `.graphboard` or `.no-data-message`?
2.  `_showComponent` is a misnomer. Sure, that is the logic you are implementing but that is not what the method is doing (it is returning a class name). Also, `_getContainerClassName` method should return a class name that describes a state instead of a generic verb, `.remove`.

Reply:
Thanks for the review! Please take another look. Thanks!